### PR TITLE
Clarify perps API considerations

### PR DESCRIPTION
**Pull request by witgaw**
Clarify data requirements related to perps.

Close https://github.com/vegaprotocol/specs/issues/1849

### CODE REVIEW

File: protocol/0053-PERP-product_builtin_perpetual_future.md [hunk 1]
```diff
@@ -190,7 +190,25 @@ Should a perpetual futures market get closed using the [governance proposal](./0
 
 ### API considerations
 
-It should be possible to query the market for the list of current funding payment data points as well as history of calculated funding payment values.
+For every completed funding period the following data should be emitted:
+
+- funding period start time,
+- funding period end time,
+- funding rate,
+- funding payment,
```
Comment (davidsiska-vega) [inline protocol/0053-PERP-product_builtin_perpetual_future.md:198]: I am assuming this is on a per-party basis?

Comment (davidsiska-vega) [reply protocol/0053-PERP-product_builtin_perpetual_future.md:198]: Or for a hypothetical position of size `1`?

File: protocol/0053-PERP-product_builtin_perpetual_future.md [hunk 2]
```diff
@@ -190,7 +190,25 @@ Should a perpetual futures market get closed using the [governance proposal](./0
...
+- external (spot) price TWAP,
+- internal (mark) price TWAP.
+
+Furthermore, within the ongoing funding period the following data should be emitted at least every time the mark price is updated:
+
+- funding period start time,
+- estimate time,
```
Comment (davidsiska-vega) [inline protocol/0053-PERP-product_builtin_perpetual_future.md:205]: This is a bit unclear to me; is it like "funding period end time", if the end was now?

**Review by davidsiska-vega — CHANGES_REQUESTED** [review]
Some comments...

File: protocol/0053-PERP-product_builtin_perpetual_future.md (hunk 2 above)
Comment (witgaw) [reply protocol/0053-PERP-product_builtin_perpetual_future.md:205]: Hopefully clarified now.

File: protocol/0053-PERP-product_builtin_perpetual_future.md (hunk 1 above)
Comment (witgaw) [reply protocol/0053-PERP-product_builtin_perpetual_future.md:198]: Yes, clarified now.

**Comment by JonRay15** [comment]
As mentioned on Slack the only thing I still dont see anywhere here is the static on the market details API related to funding duration & frequency, which we need in order to handle the countdown to the next funding event.  Would like to see this added before merging please.

**Review by davidsiska-vega — APPROVED** [review]

**Pull request merged**